### PR TITLE
main: use wayland sockets from wayland-1 to wayland-32

### LINKED ISSUE #738: Avoid wayland-0
This would be to work around libwayland-client's behavior, see https://gitlab.freedesktop.org/wayland/weston/commit/ae69381b6eb4. Relevant code in Wayfire is at: https://github.com/WayfireWM/wayfire/blob/5bdc2b20faa8897b02d7437091be52f0a6ed2174/src/main.cpp#L370-L376

### PR DESCRIPTION
**Pull request by ammen99**
Fixes #738

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -367,28 +381,28 @@ int main(int argc, char *argv[])
...
+        LOGE("Failed to initialize backend, exiting");
         wlr_backend_destroy(core.backend);
         wl_display_destroy(core.display);
 
         return -1;
     }
 
-    LOGI("running at server ", server_name);
-    setenv("WAYLAND_DISPLAY", server_name, 1);
+    setenv("_WAYLAND_DISPLAY", core.wayland_display.c_str(), 1);
```
Comment (wb9688) [inline src/main.cpp:403]: What's the point of this env var?

Comment (ammen99) [reply src/main.cpp:403]: I don't know lol, I copied that part from somewhere else and have not changed it since.

Comment (ammen99) [reply src/main.cpp:403]: Here is where this comes from: https://github.com/swaywm/rootston/blob/master/main.c#L46

However, sway doesn't set it.

Comment (ammen99) [reply src/main.cpp]: As emersion said on IRC, this is not really necessary, I removed it.

**Pull request merged**